be/src/olap/snapshot_manager.cpp

```diff
@@ -98,13 +98,13 @@ OLAPStatus SnapshotManager::release_snapshot(const string& snapshot_path) {
             continue;
         }
         std::string abs_path;
-        RETURN_WITH_WARN_IF_ERROR(store->env()->canonicalize(store->path(), &abs_path),
+        RETURN_WITH_WARN_IF_ERROR(Env::Default()->canonicalize(store->path(), &abs_path),
```

Comment:
DataDir has no env now. As remote storage is using StorageBackend, only PosixEnv left，DataDir->env() is useless.
And if data_dir is a remote_cache path, it maybe multi StorageBackend type, it can not be set as an env.